App.tsx

```diff
@@ -25,7 +27,7 @@ export default function App() {
 
   return (
     <>
-      <StatusBar style="dark" backgroundColor="#FFF" translucent />
+      <StatusBar style="dark" backgroundColor="transparent" translucent />
```

Comment:
Will all app screens have a transparent StatusBar? In my opinion, especially on screens that have scrool it's better to leave a defined color, as the StatusBar content can be without contrast with the screen content.

Reply:
![20210705_101930.jpg](https://user-images.githubusercontent.com/50719694/124477793-cf623300-dd7a-11eb-9d85-5f34edc41046.jpg)

Reply:
Good observation, I'm going to create a issue to reporting this, so we managed to organize ourselves better and leave this PR just for the splash screens

Reply:
Good observation, I think you should create an issue to report this, so we managed to organize ourselves better and leave this PR only for the initial screens